### DISCUSSION TITLE
Is it possible to import and use a route created in another app?

### DISCUSSION THREAD

**Discussion by sjsbikes** [organicmaps/organicmaps#6071] [Q&A]
GPX files can be viewed, although they can't be followed as a route?

This seems to be a major oversight, unless its already been implemented, and its a case of operator error on my part.

As an example, I create a routes on https://gpx.studio/ and import them on OSMand. (which I'd really like to replace)

**Comment by cyber-toad** [comment]
Not yet, see https://github.com/organicmaps/organicmaps/issues/1360 https://github.com/organicmaps/organicmaps/issues/2384
